src/component/mxgraph/shape/render/IconPainter.ts

```diff
@@ -583,6 +583,57 @@ export default class IconPainter {
     canvas.close();
     canvas.fillAndStroke();
   }
+
+  paintSequentialMultiInstanceIcon({ c, ratioFromParent, shape, icon }: PaintParameter): void {
+    const originalIconSize = { width: 16, height: 16 };
+    const canvas = new BpmnCanvas({
+      mxCanvas: c,
+      shapeConfiguration: shape,
+      iconConfiguration: {
+        originalSize: originalIconSize,
+        style: icon,
+        ratioFromShape: ratioFromParent,
+      },
+    });
+    canvas.setIconOriginToShapeBottomLeft();
+
+    c.setFillColor(icon.strokeColor);
```

Comment:
I tought we have a intermediate method just for the icon drawing, and rotate it if it's a sequential or a parallel, like for the gateway cross :slightly_smiling_face:

Reply:
I initially planned to do this but as the code is simple, I skipped doing it
I will review that once dependent PR are merged and this one is rebased

Reply:
hm, this doesn't work very well in this case. 
The rotation applies to the latest close path, here we have 3 paths , so only the latest rectangle is rotated.
This work well for the gateway cross because it is a single path

![image](https://user-images.githubusercontent.com/27200110/89538735-3984f780-d7fb-11ea-8b55-c676c696e716.png)